src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -164,4 +164,8 @@
     </StringProperty>
 
     <DynamicEnumProperty Name="SupportedTargetFrameworks" DisplayName="Supported TargetFrameworks" EnumProvider="SupportedTargetFrameworksEnumProvider" Visible="False"/>
+    
+    <!-- F# specific properties-->
+    <BoolProperty Name="CanUseTargetFSharpCoreVersion" Visible="False"/>
```

Comment:
So originally this rule system was implemented for C++ to show values in C++ prop pages\property grid. For managed projects we don't use the property grid but use appdesigner instead. So this makes it invisible in the property grid but makes it present in the BrowseObject. The app designer queries the browseobject directly and so doesn't see this Visible attribute.